src/main/java/seedu/duke/logic/commands/module/ListModuleCommand.java

```diff
@@ -21,12 +21,16 @@ public ListModuleCommand(String argument) {
     @Override
     public void execute(Ui ui, Storage storage, TaskList taskList, LessonList lessonList, ModuleList moduleList)
             throws DukeException, IOException {
-        assert (argument.equalsIgnoreCase("verbose") | argument.isEmpty())
-                : "list module command currently only accepts empty, \"verbose\"";
-        if (argument.equalsIgnoreCase("verbose")) {
-            ui.printModulesWithDetails(moduleList);
+        if (moduleList.isEmpty()) {
+            ui.printMessage("There are no modules in the list.");
         } else {
-            ui.printModuleList(moduleList);
+            assert (argument.equalsIgnoreCase("verbose") | argument.isEmpty())
+                    : "list module command currently only accepts empty, \"verbose\"";
```

Comment:
Faulty input should be handled with exception instead?

Reply:
This is also not written by me but yep I can change it.